packages/core/src/browser/keyboard/browser-keyboard-layout-provider.ts

```diff
@@ -33,8 +33,8 @@ export class BrowserKeyboardLayoutProvider implements KeyboardLayoutProvider, Ke
     @inject(ILogger)
     protected readonly logger: ILogger;
 
-    @inject(LocalStorageService)
-    protected readonly storageService: LocalStorageService;
+    @inject(StorageService)
+    protected readonly storageService: StorageService;
```

Comment:
Are you sure that it is correct? @spoenemann was an intent to store in local storage without a workspace prefix that it is shared between all windows regardless of a workspace

Reply:
Yes, LocalStorageService was chosen intentionally.

Reply:
I see, okay, then this PR is redundant. Going to close it.

Reply:
@vzhukovskii i think in your case maybe you need to rebind `StorageService` and `LocalStorageService` then, not sure